doc/connectivity/bluetooth/api/audio/bluetooth-le-audio-arch.rst

```diff
@@ -59,6 +59,292 @@ GAF has been implemented in Zephyr with the following structure.
 
    Zephyr Generic Audio Framework
 
+
+Dependency on GATT
+==================
+
+The following tables show the required sub-procedures from GATT for each of the
+LE Audio profiles and services.
+
+.. table:: GATT sub-procedures for LE Audio profiles
+   :widths: auto
```

Comment:
For all tables: Suggest to center the values in the cell, as the specs do: 
![image](https://github.com/zephyrproject-rtos/zephyr/assets/2520413/d2b836c0-e493-45c4-9bf6-59d47e8f5d97)

Reply:
That would be nice; it should be possible with .RST format (by adding space before and after the cell-text), but that didn't work. Leaving it as is for now.